lib/ruby_lsp/requests/formatting.rb

```diff
@@ -33,6 +33,8 @@ def run
         formatted_text = formatted_file
         return unless formatted_text
 
+        return if formatted_text.size == @document.source.size && formatted_text == @document.source
```

Comment:
Shouldn't this be using `size` declared just bellow?

Reply:
I changed to use the `size` variable, I totally had missed it.

Let's wait to move it into `Document` if the need to re-use this arises. I feel like this won't be needed elsewhere.